### PR TITLE
Update relevant external links to `<ExternalLink>`

### PR DESCRIPTION
**Pull request by rhamilto**

### CODE REVIEW

**Comment by spadgett** [comment]
We should try to switch these to the `ExternalLink` component where we can.

**Comment by rhamilto** [comment]
Updated to switch all relevant external links to use `<ExternalLink>`.  PTAnotherLook, @spadgett.

**Comment by rhamilto** [comment]
jenkins rebuild

File: frontend/public/components/utils/link.jsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ export const getNamespace = path => {
   return match && match.length > 0 && match[0];
 };
 
-export const ExternalLink = ({href, text}) => <a className="co-external-link" href={href} target="_blank" rel="noopener noreferrer">{text}</a>;
+export const ExternalLink = ({href, text, additionalClassName=''}) => <a className={`co-external-link ${additionalClassName}`} href={href} target="_blank" rel="noopener noreferrer">{text}</a>;
```
Comment (spadgett) [inline frontend/public/components/utils/link.jsx]:
```suggestion
export const ExternalLink = ({href, text, additionalClassName=''}) => <a className={classNames('co-external-link', additionalClassName)} href={href} target="_blank" rel="noopener noreferrer">{text}</a>;
```

Comment (rhamilto) [reply frontend/public/components/utils/link.jsx]: Updated

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
Rebased to pick up OLM test fixes.

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
@rhamilto I suspect the CI failure is something in your changes. I haven't seen that error in other PRs, and it failed on the same test twice.

**Comment by spadgett** [comment]
Seems to be a runtime error on the installed operators page.

https://reactjs.org/docs/error-decoder.html/?invariant=130

```
Level { name_: 'SEVERE', value_: 1000 } 'http://localhost:9000/path-D9qxBM1aqv/static/vendors~main-chunk-ddc293fe9f434294cc9f.min.js 106:53649 Error: Minified React error #130; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.\n    at f (http://localhost:9000/path-D9qxBM1aqv/static/vendors~main-chunk-ddc293fe9f434294cc9f.min.js:107:302)\n    at Kn (http://localhost:9000/path-D9qxBM1aqv/static/vendors~main-chunk-ddc293fe9f434294cc9f.min.js:107:34431)\n    at p (http://localhost:9000/path-D9qxBM1aqv/static/vendors~main-chunk-ddc293fe9f434294cc9f.min.js:107:39243)\n    at v (http://localhost:9000/path-D9qxBM1aqv/static/vendors~main-chunk-ddc293fe9f434294cc9f.min.js:107:40492)\n    at http://localhost:9000/path-D9qxBM1aqv/static/vendors~main-chunk-ddc293fe9f434294cc9f.min.js:107:42423\n    at p (http://localhost:9000/path-D9qxBM1aqv/static/vendors~main-chunk-ddc293fe9f434294cc9f.min.js:107:42714)\n    at c (http://localhost:9000/path-D9qxBM1aqv/static/vendors~main-chunk-ddc293fe9f434294cc9f.min.js:107:42651)\n    at beginWork (http://localhost:9000/path-D9qxBM1aqv/static/vendors~main-chunk-ddc293fe9f434294cc9f.min.js:107:51723)\n    at o (http://localhost:9000/path-D9qxBM1aqv/static/vendors~main-chunk-ddc293fe9f434294cc9f.min.js:107:54823)\n    at a (http://localhost:9000/path-D9qxBM1aqv/static/vendors~main-chunk-ddc293fe9f434294cc9f.min.js:107:55102)'
END BROWSER LOGS
```

**Comment by rhamilto** [comment]
I get the same error when I run locally against master.  :|

**Comment by rhamilto** [comment]
> I get the same error when I run locally against master. :|

OLM tests appear to still be flaky, but my bug is fixed.  Thanks for the heads up, @spadgett.

**Pull request merged**